### DISCUSSION TITLE
How to import "activeAt" field as a label of prometheus

### DISCUSSION THREAD

**Discussion by ghost** [prometheus/prometheus#10077] [Q&A] [answered]
I want to import the "activeAt" field as a label of prometheus, after I  configure it like 
_alert: test
expr: xxx
labels:
   time: '{{  $activeAt }}'_

It gives me undefined variable \"$activeAt\"" error.

**Comment by codesome** [comment, marked as answer]
`activeAt` is not available for templating. Entire list of supported labels and functions is here https://prometheus.io/docs/prometheus/latest/configuration/template_reference/
